cves/kernel/CVE-2021-29647.yml

```diff
@@ -253,8 +246,8 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: This feature does not impact any internationalization features.
```

Comment:
Why not? You should explain why internationalization is not relevant here instead of simply stating that it isn't